apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -103,7 +103,6 @@ type TempoStackSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Tempo Component Templates"
 	Template TempoTemplateSpec `json:"template,omitempty"`
 
-	// NOTE: currently this field is not considered.
```

Comment:
cc @rubenvp8510 this note is outdated, right? The setting of this field lands in the tempo config:
```
    ingester:
      lifecycler:
        ring:
          replication_factor: 2
```

Reply:
Yes it is outdated, I might forgot to remove it when this was implemented.